Runtime/MeshOperations/AppendElements.cs

```diff
@@ -47,12 +54,18 @@ public static class AppendElements
 
             var mc = mesh.HasArrays(MeshArrays.Color);
             var fc = colors != null;
-            var mt = mesh.HasArrays(MeshArrays.Texture0);
-            var ft = uvs != null;
+            var mt0 = mesh.HasArrays(MeshArrays.Texture0);
+            var ft0 = uv0s != null;
+			var mt2 = mesh.HasArrays(MeshArrays.Texture2);
+            var ft2 = uv2s != null;
+			var mt3 = mesh.HasArrays(MeshArrays.Texture3);
+            var ft3 = uv3s != null;
 
             Vector3[] newPositions = new Vector3[vertexCount + faceVertexCount];
             Color[] newColors = (mc || fc) ? new Color[vertexCount + faceVertexCount] : null;
-            Vector2[] newTextures = (mt || ft) ? new Vector2[vertexCount + faceVertexCount] : null;
+            Vector2[] newTexture0s = (mt0 || ft0) ? new Vector2[vertexCount + faceVertexCount] : null;
+            Vector4[] newTexture2s = (mt2 || ft2) ? new Vector4[vertexCount + faceVertexCount] : null;
+            Vector4[] newTexture3s = (mt3 || ft3) ? new Vector4[vertexCount + faceVertexCount] : null;
```

Comment:
Since these texture channels are already stored as lists, could we not just append to the existing values?